teuthology/orchestra/run.py

```diff
@@ -90,8 +90,9 @@ def execute(self):
         prefix = "Running:"
         if self.label:
             prefix = "Running ({label}):".format(label=self.label)
-        log.getChild(self.hostname).info(u"{prefix} {cmd!r}".format(
-            cmd=self.command, prefix=prefix))
+        log.getChild(self.hostname).info(prefix)
+        for line in self.command.split('\n'):
+            log.getChild(self.hostname).info(line)
```

Comment:
Nice, this is what I meant. However, since there is used _logger info_, and not _print()_ used it is probably better to prefix each line of command with specially character to give the user an easy way to recognize a script, for example, '> '.

Reply:
@kshtsk updated and repushed.